recipes/eland/run_test.py

```diff
@@ -0,0 +1,11 @@
+from eland import (
```

Comment:
I don't think you need the `run_test.py` file, since the import tests in the `meta.yaml` does the same thing